src/api/sloppy.go

```diff
@@ -53,9 +57,7 @@ type Client struct {
 // NewClient returns a new Client to handle API requests.
 func NewClient() *Client {
 	client := &Client{
-		client: &http.Client{
-			Timeout: 2 * time.Minute,
-		},
+		client:    http.DefaultClient,
```

Comment:
Yes, the `DefaultClient`has no timeout because `var DefaultClient = &Client{}` and a timeout of zero means no timeout (https://golang.org/src/net/http/client.go#L69).